Scenario5/Scenario5/Program.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Scenario5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Account johnAccount = GetAccount();
+            Console.WriteLine(SerializeToCustomJson(johnAccount));
+
+            string customJson = GetCustomFormattedAccountJson();
+            Account jetAccount = DeserializeFromCustomJson(customJson);
+            Console.WriteLine(jetAccount?.Email);
+            Console.WriteLine(jetAccount?.CreatedDate);
+
+            Console.WriteLine("Press any key to continue ...");
+            Console.ReadLine();
+        }
+
+        // The Example Company uses the MM/dd/yy format for DateTimeOffset values in their Account models.
+        // TODO: 1) Serialize the "account" object to a custom-formatted JSON string and return it.
+        private static string SerializeToCustomJson(Account account)
+        {
+            // <Add/modify code here>
+            return "";
+        }
+
+        // The Example Company uses the MM/dd/yy format for DateTimeOffset values in their Account models.
+        // TODO: 2) Deserialize the custom-formatted JSON string as an "account" object and return it.
+        private static Account DeserializeFromCustomJson(string json)
+        {
+            // <Add/modify code here>
+            return null;
+        }
+
+        private static Account GetAccount()
+        {
+            // Note: Do NOT modify the Account object creation.
+            Account account = new Account
+            {
+                Email = "john@example.com",
+                CreatedDate = DateTimeOffset.Now
```

Comment:
Don't use Now. Each subsequent run when user will be debugging will change the output and that might confuse them.

Reply:
A constant works:
```suggestion
                CreatedDate = new DateTimeOffset(2019, 8, 31);
```